### PR TITLE
Support multiple ICD Versions

### PR DESCRIPTION
**Pull request by noemide**
In Condition and SampleDiagnosis Extension, make it possible to use ICD-10-WHO, ICD-10-GM and ICD-9 (also in different year versions).

### CODE REVIEW

File: terminology/valueset/SampleContentDiagnosisICD-10-GM.ValueSet.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
   "resourceType": "ValueSet",
-  "url": "https://fhir.bbmri.de/ValueSet/SampleContentDiagnosis",
-  "name": "SampleContentDiagnosis",
-  "title": "sample content diagnosis",
+  "url": "https://fhir.bbmri.de/ValueSet/SampleContentDiagnosisICD-10-GM",
+  "name": "SampleContentDiagnosisICD-10-GM",
+  "title": "sample content diagnosis ICD-10-GM",
```
Comment (alexanderkiel) [inline terminology/valueset/SampleContentDiagnosisICD-10-GM.ValueSet.json:5]: here you should use the german system URL

File: terminology/valueset/SampleContentDiagnosisICD-9.ValueSet.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    "include": [
+      {
+        "system": "https://fhir.bbmri.de/CodeSystem/OtherSampleDiagnosis"
+      },
+      {
+        "system": "http://hl7.org/fhir/sid/icd9"
+      }
+    ]
+  }
+}
```
Comment (alexanderkiel) [inline terminology/valueset/SampleContentDiagnosisICD-9.ValueSet.json]: newline :-)

**Review by alexanderkiel — CHANGES_REQUESTED** [review]
Is it possible that the validator handles the other codes for SampleContentDiagnosis? Is yes, we could add such an extension to the examples.

File: terminology/valueset/SampleContentDiagnosisICD-9.ValueSet.json (hunk 2 above)
Comment (noemide) [reply terminology/valueset/SampleContentDiagnosisICD-9.ValueSet.json]: Done!

**Comment by noemide** [comment]
> 
> 
> Is it possible that the validator handles the other codes for SampleContentDiagnosis? Is yes, we could add such an extension to the examples.

Is now adressed in #29 .

**Review by alexanderkiel — APPROVED** [review]
Please squash at least the fixing commits before merging.

**Pull request merged**